site/sdk/ts-guide.md

```diff
@@ -0,0 +1,245 @@
+# Typescript SDK Developer Guide
+
+This guide will walk you through developing a kpt function using the Typescript SDK.
+
+## Setup
+
+### System Requirements
+
+Currently supported platforms: amd64 Linux/Mac
```

Comment:
generally JS is fairly cross platform.  amd64 is fairly restrictive, any reason for that?

Reply:
Node itself is not the issue, it's the Go binary dependencies: kpt and the generator binary.